website/docs/d/vra_region_enumeration_aws.html.markdown

```diff
@@ -0,0 +1,28 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration_aws"
+description: |-
+  Provides a data lookup for vra_region_enumeration_aws.
+---
+
+# Data Source: vra_region_enumeration_aws
+## Example Usages
+
+This is an example of how to read a region enumeration data source for AWS cloud account.
+
+**Region enumeration vSphere data source by its id:**
+```hcl
+data "vra_region_enumeration_aws" "this" {
+  id = vra_region_enumeration_aws.this.id
+}
+```
```

Comment:
Change the title to reflect `region_enumeration_aws` data source.

Example is incorrect as well. You must need `access_key` and `secret_key` arguments for this data source.